.github/workflows/migrations.yml

```diff
@@ -1,99 +1,65 @@
-name: migrations
+name: generate migration sql
 on:
   pull_request:
     paths:
       - 'src/sentry/migrations/*'
 
 jobs:
-    sql:
-      name: Generate SQL
-      runs-on: ubuntu-16.04
+  task:
```

Comment:
I'm guessing you're trying to signal here that this is a single-job workflow. If so I'd name this `main`.